examples/applications/plot_stock_market.py

```diff
@@ -85,8 +85,6 @@
 # that we get high-tech firms, and before the 2008 crash). This kind of
 # historical data can be obtained for from APIs like the quandl.com and
 # alphavantage.co ones.
```

Comment:
The comments above is necessary?

Reply:
I would keep it because it explains a bit were the data is coming from and how you get such data if you wanted to add other stocks.